test/common/index.js

```diff
@@ -271,9 +271,24 @@ Object.defineProperty(exports, 'hasFipsCrypto', {
 });
 
 {
-  const localRelative = path.relative(process.cwd(), `${exports.tmpDir}/`);
-  const pipePrefix = exports.isWindows ? '\\\\.\\pipe\\' : localRelative;
-  const pipeName = `node-test.${process.pid}.sock`;
+  let pipePrefix;
+  if (exports.isWindows) {
+    pipePrefix = '\\\\.\\pipe\\';
+  } else {
+    const localRelative = path.relative(process.cwd(), `${exports.tmpDir}`);
+    pipePrefix = localRelative + path.sep;
+  }
+
+  let pipeName;
+  if (require.main) {
+    const filename = path.basename(require.main.filename);
```

Comment:
I'd say go K.I.S.S. Since this only needs to be one way random, just take `Math.random`

Reply:
Good idea

Reply:
Also, then we don't need the pid anymore...